ydb/src/connection_pool.rs

```diff
@@ -1,20 +1,37 @@
 use crate::YdbResult;
 use http::Uri;
+use tracing::trace;
 use std::collections::HashMap;
 use std::sync::{Arc, Mutex};
 use std::time::{Duration, Instant};
 use http::uri::Scheme;
-use tonic::transport::{Channel, ClientTlsConfig, Endpoint};
+use tonic::transport::{Certificate, Channel, ClientTlsConfig, Endpoint};
 
 #[derive(Clone)]
 pub(crate) struct ConnectionPool {
     state: Arc<Mutex<ConnectionPoolState>>,
+    tls_config: Arc<Option<ClientTlsConfig>>,
 }
 
 impl ConnectionPool {
     pub(crate) fn new() -> Self {
         Self {
             state: Arc::new(Mutex::new(ConnectionPoolState::new())),
+            tls_config: None.into(),
+        }
+    }
+
+    pub(crate) fn load_certificate(self, path: String) -> Self {
+        trace!("path is {}", path);
```

Comment:
better merge it to one record. At real world system can generate a lot of logs even at start (with multiply treads) and two records may be separated by many other records if trace write as logs.